### ISSUE TITLE
🐛 Bug: Cannot create a formula in a related child table using a DisplayValue field where contents of DisplayValue field from Parent Table is derived from a formula

### ISSUE THREAD

**Issue by ibarot1981** [nocodb/nocodb#5208]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Table MaterialSize has a DisplayValue field called MaterialTypeSize which is a Formula field. 
Formula is a CONCAT({MaterialType}, " ", {Size}). 

Table MaterialMaster has many to one relationship with Table MaterialSize. So, by setting MaterialTypeSize as DisplayValue, I get the desired dropdown to enter records in MaterialMaster. In MaterialMaster this field shows up as MaterialSize which I believe is desired behaviour. I get the dropdowns which are the concatenated values from MaterialSize table.

Issue occurs when I use this field MaterialSize in MaterialMaster in a formula, it does not allow me to reference it at all and gives me following error in container logs :

Error: Formula error: select CONCAT(IFNULL((select MaterialTypeSize from MaterialSize where MaterialSize.Size_ID = MaterialMaster.Size_ID), ''),IFNULL(' - ', ''),IFNULL(Material_Desc, '')) from MaterialMaster - Unknown column 'MaterialTypeSize' in 'field list'

So, basically I am not able to use the DisplayValue field, which is a formula field itself in parent table,  as a formula field in child table. 

Sharing ERD of my project here below. 

### Desired Behavior

I should be able to use the field in formula

### Project Details

Node: v16.19.1
Arch: x64
Platform: linux
Docker: true
RootDB: mysql2
PackageVersion: 0.105.3

Created new Project by connecting to external DB. Connected to MariaDB running in a docker container

### Attachments

![ERD](https://user-images.githubusercontent.com/102347724/221354768-266d944b-37ee-40b9-9a5a-f16a77c418a7.jpg)

**Comment by wunter8** [comment]
I've had the same issue

**Comment by dannyge** [comment]
Same here!

**Comment by fendy3002** [comment]
this should be fixed already with the latest release
